src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -0,0 +1,171 @@
+import math
+from collections import defaultdict
+from datetime import datetime
+
+import sentry_sdk
+from django.utils import timezone
+
+from sentry import features
+from sentry.constants import DataCategory
+from sentry.models.activity import Activity
+from sentry.models.group import Group
+from sentry.models.organization import Organization, OrganizationStatus
+from sentry.models.release import Release, ReleaseProject
+from sentry.services.hybrid_cloud.user_option import user_option_service
+from sentry.silo import SiloMode
+from sentry.snuba.referrer import Referrer
+from sentry.tasks.base import instrumented_task, retry
+from sentry.tasks.summaries.weekly_reports import (
+    ONE_DAY,
+    OrganizationReportContext,
+    fetch_key_error_groups,
+    fetch_key_performance_issue_groups,
+    project_event_counts_for_organization,
+    project_key_errors,
+    project_key_performance_issues,
+    user_project_ownership,
+)
+from sentry.types.activity import ActivityType
+from sentry.types.group import GroupSubStatus
+from sentry.utils.dates import floor_to_utc_day, to_datetime, to_timestamp
+from sentry.utils.outcomes import Outcome
+from sentry.utils.query import RangeQuerySetWrapper
+
+
+# The entry point. This task is scheduled to run every day at 4pm PST.
+@instrumented_task(
+    name="sentry.tasks.daily_summary.schedule_organizations",
+    queue="reports.prepare",
+    max_retries=5,
+    acks_late=True,
+    silo_mode=SiloMode.REGION,
+)
+@retry
+def schedule_organizations(
```

Comment:
I'm kinda torn between leaving this in and taking it out for the next PR that will actually send a message. This is weird to test as is because it doesn't send anything. If I remove this for this PR, I'd remove the celery task and feature flag as well.

Reply:
Is this a placeholder to manually execute instead of waiting for the cron? (i think that'd be okay to leave)